packages/core/src/item.ts

```diff
@@ -318,9 +332,13 @@ export function guessFieldType({
         return matchedTypeByName;
     }
 
-    const matchedTypeByValue = Object.keys(FIELD_DEFS).filter((fieldType) =>
-        FIELD_DEFS[fieldType].pattern.test(value)
-    )[0] as FieldType;
+    // We skip some because they can match anything, and are only really valuable when matched by name
+    const fieldTypesToSkipByValue = [FieldType.Username, FieldType.Password];
```

Comment:
If I didn't skip these, it would always be matched as them because they allow anything.